src/sagas/networkSettings.js

```diff
@@ -41,7 +40,17 @@ export function* changeNetworkSettings({ data, pin }) {
 
   let versionData;
   try {
-    versionData = yield call([wallet, wallet.getVersionData]);
+    const versionUrl = new URL('version', data.node).toString();
+    /*
+     * Using fetch instead of the wallet-lib axios instance to bypass
+     * the axios interceptor that is used to check for network errors.
+     * If this request fails, we don't want the network error modal to be shown.
+     */
+    const response = yield call(fetch, versionUrl);
```

Comment:
question(non-blocking): Given that we've found the problem with Axios ( the same solved by https://github.com/HathorNetwork/hathor-explorer/pull/404 ), would it be a good idea to use a "blank" `axios` instance here instead of using `fetch`?

That way, we would bypass the lib interceptor, but keep the syntax that is already used throughout the app already.

Reply:
We don't use axios in the wallet-desktop except for the `axiosInstance` created from the wallet-lib (which is exactly what we don't want to do here)

So I prefer just using `fetch`, which is native since this is a simple http request